### PR TITLE
create CONTRIBUTING.portuguese.md

### PR DESCRIPTION
**Pull request by fi-carvalho**
Created portuguese version of CONTRIBUTING

### Affected components 

- Server
- Console
- CLI
- Docs
- Community Content
- Build System
- Tests
- Other (list it)

### CODE REVIEW

**Comment by fi-carvalho** [comment]
Created portuguese version of CONTRIBUTING.md

File: translations/CONTRIBUTING.portuguese.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- Não use maiúscula na primeira letra na linha do sumário.
+
+- Não coloque ponto (.) ao final da linha do sumário.
+
+## Traduções
+
+Esse documento está disponível em:
+
+- [French :fr:](translations/CONTRIBUTING.french.md)
```
Comment (marionschleifer) [inline translations/CONTRIBUTING.portuguese.md]: Can you add the English one as well, please?

Comment (marionschleifer) [reply translations/CONTRIBUTING.portuguese.md]: Also, can you add the link to this Portuguese translation to the English `CONTRIBUTING.md` please?

**Comment by marionschleifer** [comment]
@fi-carvalho I'm closing this PR. If you would like to implement the suggested changes, feel free to re-open 🙂

**Pull request closed without merging**